docs/04-sketch-design-system.md

```diff
@@ -1,47 +0,0 @@
-## Sketch Design System & Library
```

Comment:
I opted to remove this since the best place to keep the instructions for setup is now in the Figma "Getting Started" page (which I updated and reference in the "Overview" page of design.stjude.cloud.